src/vue-components/inflections-table-wide.vue

```diff
@@ -103,21 +105,23 @@
         },
         classes: {
           fullMorphologyMatch: 'infl-cell--morph-match'
+        },
+        options: {
+          emptyColumnsHidden: true,
+          noSuffixMatchesHidden: true
         }
       }
     },
 
     methods: {
       initView: function () {
-        if (this.view.isRenderable) {
-          // Rendering is not required for component-enabled views
-          this.view.render()
-        }
-        this.state.noSuffixGroupsHidden = this.view.isNoSuffixMatchesGroupsHidden
-        this.$emit('widthchange')
+        // this.state.noSuffixGroupsHidden = this.view.isNoSuffixMatchesGroupsHidden
```

Comment:
are there sizing consequence to not emitting the widthchange event here?

Reply:
We need to emit that when, I believe, we render a table within this component and parent does not know about that. For lexical query tables, those are rendered within `inflections.vue` and we don't need to emit it here. Here we render inflection browser tables only, and this is done within `collapse()` where we do emit a `widthchange`.

Reply:
ok, thanks